src/docs/asciidoc/web.adoc

```diff
@@ -1,25 +1,30 @@
 [[spring-web]]
-= Web
+= The Web
```

Comment:
Documentation has been splitted between Web and Reactive Web stack, so not sure about "The web" which was previous unique title for both stack.